### ISSUE TITLE
Vemto from CLI : ENOENT: no such file or directory, open 'C:/laragon/www/testfromcmd/package.json'

### ISSUE THREAD

**Issue by respodeveloper** [VemtoOrg/vemto2-issues#51]
**Error Message:**
ENOENT: no such file or directory, open 'C:/laragon/www/testfromcmd/package.json'

**Stack:**
```
Error: ENOENT: no such file or directory, open 'C:/laragon/www/testfromcmd/package.json'
    at Object.openSync (node:fs:585:3)
    at Object.func [as openSync] (node:electron/js2c/asar_bundle:5:1812)
    at Object.readFileSync (node:fs:453:35)
    at e.readFileSync (node:electron/js2c/asar_bundle:5:9160)
    at $p.readFile (C:\Program Files\Vemto\resources\app.asar\build\src\main\main.js:1:5015)
    at C:\Program Files\Vemto\resources\app.asar\build\src\main\main.js:16:5792
    at C:\Program Files\Vemto\resources\app.asar\build\src\main\main.js:1:6782
    at Generator.next (<anonymous>)
    at C:\Program Files\Vemto\resources\app.asar\build\src\main\main.js:1:6611
    at new Promise (<anonymous>)
```

CLI Log

```
C:\Users\bongani>vemto.exe

C:\Users\bongani>
Checking for update
Update for version 2.0.6 is not available (latest version: 2.0.6, downgrade is disallowed).
checkForUpdatesAndNotify called, downloadPromise is null
(stderr) FAILED to execute command: composer create-project laravel/laravel:^11.0 testfromcmd

Creating a "laravel/laravel:11.0" project at "./testfromcmd"
Cannot use laravel/laravel's latest version v11.0.0 as it requires php ^8.2 which is not satisfied by your platform.

In CreateProjectCommand.php line 421:

  Could not find package laravel/laravel with version 11.0 in a version installable using your PHP version, PHP exten
  sions and Composer version.


create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--no-audit] [--audit-format AUDIT-FORMAT] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--ask] [--] [<package> [<directory> [<version>]]]


Creating a "laravel/laravel:11.0" project at "./testfromcmd"
Cannot use laravel/laravel's latest version v11.0.0 as it requires php ^8.2 which is not satisfied by your platform.

In CreateProjectCommand.php line 421:

  Could not find package laravel/laravel with version 11.0 in a version installable using your PHP version, PHP exten
  sions and Composer version.


create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--no-audit] [--audit-format AUDIT-FORMAT] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--ask] [--] [<package> [<directory> [<version>]]]


Error: Error: Command failed: composer create-project laravel/laravel:^11.0 testfromcmd
Creating a "laravel/laravel:11.0" project at "./testfromcmd"
Cannot use laravel/laravel's latest version v11.0.0 as it requires php ^8.2 which is not satisfied by your platform.

In CreateProjectCommand.php line 421:

  Could not find package laravel/laravel with version 11.0 in a version installable using your PHP version, PHP exten
  sions and Composer version.


create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--no-audit] [--audit-format AUDIT-FORMAT] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--ask] [--] [<package> [<directory> [<version>]]]


Stdout:
(error) FAILED to execute command: composer create-project laravel/laravel:^11.0 testfromcmd

Error: Command failed: composer create-project laravel/laravel:^11.0 testfromcmd
Creating a "laravel/laravel:11.0" project at "./testfromcmd"
Cannot use laravel/laravel's latest version v11.0.0 as it requires php ^8.2 which is not satisfied by your platform.

In CreateProjectCommand.php line 421:

  Could not find package laravel/laravel with version 11.0 in a version installable using your PHP version, PHP exten
  sions and Composer version.


create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--no-audit] [--audit-format AUDIT-FORMAT] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--ask] [--] [<package> [<directory> [<version>]]]


    at ChildProcess.exithandler (node:child_process:407:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1092:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5) {
  code: 1,
  killed: false,
  signal: null,
  cmd: 'composer create-project laravel/laravel:^11.0 testfromcmd'
}
(error) FAILED to execute command: "php" artisan storage:link

Error: spawn C:\WINDOWS\system32\cmd.exe ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:478:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\WINDOWS\\system32\\cmd.exe',
  path: 'C:\\WINDOWS\\system32\\cmd.exe',
  spawnargs: [ '/d', '/s', '/c', '""php" artisan storage:link"' ],
  cmd: '"php" artisan storage:link'
}
(error) FAILED to execute command: composer require laravel/jetstream --no-interaction

Error: spawn C:\WINDOWS\system32\cmd.exe ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:478:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\WINDOWS\\system32\\cmd.exe',
  path: 'C:\\WINDOWS\\system32\\cmd.exe',
  spawnargs: [
    '/d',
    '/s',
    '/c',
    '"composer require laravel/jetstream --no-interaction"'
  ],
  cmd: 'composer require laravel/jetstream --no-interaction'
}
(error) FAILED to execute command: "php" artisan jetstream:install livewire --no-interaction

Error: spawn C:\WINDOWS\system32\cmd.exe ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:478:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\WINDOWS\\system32\\cmd.exe',
  path: 'C:\\WINDOWS\\system32\\cmd.exe',
  spawnargs: [
    '/d',
    '/s',
    '/c',
    '""php" artisan jetstream:install livewire --no-interaction"'
  ],
  cmd: '"php" artisan jetstream:install livewire --no-interaction'
}
Writing File: C:/laragon/www/testfromcmd/.env

```

---

Vemto 2.0.6
win32 10.0.19045
Locale: en-US
App version: 2.0.6

**Comment by Akalanka1337** [comment]
Solution: I upgraded my PHP version to 8.2.

**Comment by TiagoSilvaPereira** [comment]
Reopening as we need to improve this workflow

**Comment by TiagoSilvaPereira** [comment]
Fixed on Vemto 2.0.7
